src/Files.App/MainWindow.xaml.cs

```diff
@@ -175,6 +176,7 @@ public async Task InitializeApplicationAsync(object activatedEventArgs)
 					}
 					for (; index < fileArgs.Files.Count; index++)
 					{
+						InteropHelpers.SwitchToThisWindow(WindowHandle, true);
```

Comment:
```suggestion
					else
						InteropHelpers.SwitchToThisWindow(WindowHandle, true);
					for (; index < fileArgs.Files.Count; index++)
					{
```
One call to `SwitchToThisWindow()` is enough.

Reply:
Should I remove the others ones as well?